module/rnaseq_de.qmd

```diff
@@ -0,0 +1,78 @@
+
+It is important to note that researchers are **not** suggesting that ASD is entirely induced by gut bacteria. There is a strong genetic component to ASD. Scientists have known for years that there are both genetic and environmental components to the development or severity of **some** ASD symptoms. This research explores one possible environmental component.
+
+You can read the original research paper [here](https://pmc.ncbi.nlm.nih.gov/articles/PMC6993574/pdf/nihms-1528786.pdf).
+
+## What comparisons can students make?
+In this dataset, students have the option to look at gene expression in control vs ASD mice, as well as gene expression in both striatum and prefrontal cortex. All the mice were male and sacrificed at the same age (45 days).
+
+Additionally, students can look at gene expression in striatum vs prefrontal cortex in only ASD mice or only control mice. Likewise, students also have the option of looking at gene expression in ASD and control mice, focusing only on striatum or only on prefrontal cortex.
+
+## What does the student guide include?
+
+The student guide includes some background information that may help your students understand the sorts of questions they may want to ask with this dataset. 
+
+### Study overview and definitions
+
+In addition to the basic overview of the study above, the student guide also includes definitions of Autism Spectrum Disorder and explanations of the parts of the brain sampled in this study. It also includes a link to the [story of Phineas Gage](https://pmc.ncbi.nlm.nih.gov/articles/PMC1114479/), a railroad worker who survived an iron rod through his forehead, and whose injury led to breakthroughs in our understanding of neuroanatomy. 
+
+### The mouse as a model organism
+
+Students may not be familiar with why the mouse is used as a model organism. This section provides a very general explanation of the role of mice in laboratory research, as well as why mice can be used as models for human disease.
+
+### Learning about mouse genes
+
+This section of the student guide includes information on how to navigate the [Mouse Genome Informatics](https://informatics.jax.org/) database maintained by The Jackson Laboratory. It also has a brief description on interpreting a mouse gene ID code and non-genes that have associated IDs.
+
+
+## Datasets
```

Comment:
should this heading fall farther down? Feels like the student guide shouldnt be lumped in with datasets.

Reply:
Yeah, that makes sense